PhysicsTools/HeppyCore/python/framework/looper.py

```diff
@@ -280,6 +282,22 @@ def write(self):
         comp = pickle.load( pckfile )
         cfg.config.components=[comp]
         events_class = cfg.config.events_class
+    elif len(sys.argv) == 4 :
```

Comment:
as we discussed a few days ago, I think it's better to make it a keyword argument rather than a positional argument